blackboxopt/optimizers/testing.py

```diff
@@ -150,15 +150,20 @@ def is_deterministic_with_fixed_seed_and_larger_space(
             optimizer_class,
             optimizer_kwargs,
             objective=Objective("loss", False),
-            objectives=[Objective("loss", False)],
+            objectives=[Objective("loss", False), Objective("score", True)],
             seed=seed or 42,
         )
 
         for i in range(n_evaluations):
             es = opt.generate_evaluation_specification()
+
+            objectives = {"loss": losses[i]}
+            if isinstance(opt, MultiObjectiveOptimizer):
```

Comment:
I believe the motivation was to be sure, that MO optimizers could handle more than one objective, was not it? And on my point of view this extension makes a lot of sense. Why are you uncertain?

Reply:
Mainly because it was already a couple weeks ago, and I wasn't sure if we landed on some other solution in the meantime, rendering this irrelevant.

Reply:
Specifically, I see that we have `optimizer_kwargs` in `main`, which reads to me like this is another solution attempt:
https://github.com/boschresearch/blackboxopt/blob/main/blackboxopt/optimizers/testing.py#L133-L135

Reply:
Then i believe we've covered MOOptimizers in testing already. Let's then forget about this PR.